lib/matplotlib/axes/_axes.py

```diff
@@ -5637,8 +5635,10 @@ def pcolormesh(self, *args, **kwargs):
         miny = np.amin(Y)
         maxy = np.amax(Y)
 
-        corners = (minx, miny), (maxx, maxy)
```

Comment:
Same comment as above (keep definition of {min,max}{x,y} together with corner); not repeating this comment if it still occurs below.